packages/inter-protocol/src/proposals/upgrade-vaults.js

```diff
@@ -76,27 +48,13 @@ export const upgradeVaults = async (powers, { options }) => {
    *   Installation<import('../../src/vaultFactory/vaultFactory.js')['start']>
    * >}
    */
-  let installationP;
+  const installationP = E(zoe).installBundleID(bundleID);
+  produceVaultInstallation.reset();
+  produceVaultInstallation.resolve(installationP);
 
   await auctionsUpgradeComplete;
   auctionsUpgradeCompleteProducer.reset();
```

Comment:
It's a little surprising that the vaults upgrade is producing `auctionsUpgradeComplete`. Now I remember: this is why. A bit more docs might be nice.

Reply:
I think your earlier suggestion of a naming convention is good.

My theory here is that names like *fooProcessCompletion* should be considered to be single-use flags and should be cleaned up once consumed.